README.md

```diff
@@ -73,6 +83,12 @@ go run main.go S3.8 -profile <PROFILE> -region <REGION> [OPTIONAL_FLAGS]
 
 ## EC2.2
 
+### Usage
+
+```bash
+fsbp-fix EC2.2 -profile <PROFILE> -region <REGION> [OPTIONAL_FLAGS]
```

Comment:
Does EC2.2 / S3.8 need to be capitalised?

Reply:
probably not, though I think that's beyond the scope of this PR as it'd be a breaking change for the tool